querydsl-sql/src/main/java/com/querydsl/sql/postgresql/PostgreSQLQuery.java

```diff
@@ -52,16 +52,6 @@ public PostgreSQLQuery(Connection conn, Configuration configuration, QueryMetada
     }
 
     /**
-     * FOR SHARE causes the rows retrieved by the SELECT statement to be locked as though for update.
-     *
-     * @return the current object
-     */
-    public PostgreSQLQuery<T> forShare() {
```

Comment:
The code remains compatible because
```PostgreSQLQuery<T> extends AbstractSQLQuery<T, PostgreSQLQuery<T>>```

And now we have the ```forShare()``` function in ```AbstractSQLQuery```.